src/main/java/Deadline.java

```diff
@@ -0,0 +1,20 @@
+public class Deadline extends Task {
```

Comment:
code is very neat, strings can be declared in a single or fewer lines but this is more readable.